Content.Server/Database/ServerDbPostgres.cs

```diff
@@ -98,7 +99,8 @@ public sealed class ServerDbPostgres : ServerDbBase
             await using var db = await GetDbImpl();
 
             var exempt = await GetBanExemptionCore(db, userId);
-            var query = MakeBanLookupQuery(address, userId, hwId, db, includeUnbanned, exempt);
+            var newPlayer = !await db.PgDbContext.Player.AnyAsync(p => p.UserId == userId);
```

Comment:
Pull this out into a shared function to avoid duplicating it with the other one in this file.